examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -113,16 +114,17 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
             backgroundColor: CupertinoColors.white,
             onSelectedItemChanged: (int index) {
               setState(() {
-                _selectedMinute = index;
+                _selectedMinute = index % 60;
               });
             },
-            children: new List<Widget>.generate(60, (int index) {
+            itemBuilder: (BuildContext context, int index) {
+              final int modIndex = index % 60;
               return new Container(
                 alignment: Alignment.centerLeft,
                 padding: const EdgeInsets.only(left: 32.0),
-                child: new Text(index.toString()),
+                child: new Text(modIndex.toString()),
```

Comment:
this probably needs to go through the localization machinery

Reply:
This demo is only temporary until I implement the DatePicker, localization will be taken care of there.